src/site/content/en/metrics/inp/index.md

```diff
@@ -44,20 +44,88 @@ This article explains how INP works, how to measure it, and offers advice for im
 
 ## What is INP?
 
-INP aims to represent a page's overall responsiveness by measuring all click, tap, and keyboard interactions made with a page. The longest of those observed interactions&mdash;with some exceptions noted below&mdash;is chosen as the page's INP value when the user is done with the page.
+INP aims to represent a page's overall responsiveness. It does so by measuring every click, tap, and keyboard interaction made with a page.
+
+### What is an interaction?
+
+An interaction is one or more discrete user input actions. More than one interaction may consist of what's considered a "logical user gesture". For a key board interaction, this would consist of both a `keydown` _and_ a `keyup` event.
+
+### What is an event?
+
+Interactions are driven by events. Events fire callbacks during the same logical user gesture. For example, "tap" interactions on a touchscreen device include multiple events, such as `pointerup`, `pointerdown`, and `click`. While JavaScript is a common driver of interactivity, interactions can also be driven by CSS and built-in browser controls (such as `<input>` elements, for example).
```

Comment:
"Driver of interactivity" sounds like initiator, rather than handler.

WDYT about this instead?

```suggestion
Interactions are driven by events. Events fire callbacks during the same logical user gesture. For example, "tap" interactions on a touchscreen device include multiple events, such as `pointerup`, `pointerdown`, and `click`. While JavaScript is a common handler of interactivity, interactions can also be handled by CSS and built-in browser controls (such as `<input>` elements, for example).
```